src/rust/ensogl/src/animation/physics/inertia.rs

```diff
@@ -29,6 +29,66 @@ pub trait Value
 // === Properties ===
 // ==================
 
+macro_rules! define_opr_mods {
```

Comment:
Not sure what the exact intention is — but if this is "physics" and I see "Mass" — then mass divided by mass should not be mass anymore but just a scalar. Similar issue with multiplication.
(disregard if this is not about units)

Reply:
such error is not present there but as we discussed via phone, its possible to do float/mass now. fixing